met-web/src/components/landing/LandingComponent.tsx

```diff
@@ -120,11 +120,11 @@ const LandingComponent = () => {
                         ref={tileBlockRef}
                     >
                         <Grid item xs={12} sm={6} md={4} lg={4}>
-                            <MetLabel sx={{ marginBottom: '2px', display: 'flex' }}>Engagment name</MetLabel>
+                            <MetLabel sx={{ marginBottom: '2px', display: 'flex' }}>Engagement name</MetLabel>
                             <Autocomplete
                                 options={engagementOptions || []}
                                 onInputChange={(_event, newInputValue) => {
-                                    debounceLoadEngagments(newInputValue);
+                                    debounceLoadEngagements(newInputValue);
```

Comment:
good job in renaming... leave the code in a better place than you found it..appreciate it